tests/OperationTest.php

```diff
@@ -56,23 +56,75 @@ public function it_can_cancel_itself()
     /** @test */
     public function it_can_schedule_itself()
     {
-        ExampleOperation::schedule(Carbon::now()->addDay());
+        $operation = ExampleOperation::schedule(Carbon::now()->addDay());
 
-        $operation = ExampleOperation::first();
         $this->assertEquals(Carbon::now()->addDay(), $operation->should_run_at);
     }
 
     /** @test */
     public function it_can_schedule_itself_with_custom_columns()
     {
-        ColumnOperation::schedule(Carbon::now()->addDay(), [
+        $operation = ColumnOperation::schedule(Carbon::now()->addDay(), [
             'value' => 2500,
             'message' => 'Custom Columns',
         ]);
 
-        $operation = ColumnOperation::first();
         $this->assertEquals(Carbon::now()->addDay(), $operation->should_run_at);
         $this->assertEquals(2500, $operation->value);
         $this->assertEquals('Custom Columns', $operation->message);
     }
+
+    /** @test */
+    public function it_can_dispatch_itself_which_schedules_it_to_run_now()
+    {
+        $operation = ExampleOperation::dispatch();
+
+        $this->assertEquals(Carbon::now(), $operation->should_run_at);
```

Comment:
is `Carbon::now()` atomic?  Or could there be a time when the system clock turns over between the time the value is set and the assertion is done?
Should we use the method to setNow just in case?

Reply:
I actually added a call to setTestNow when I first wrote this test, but remembered that because of how many times I run into issues with carbon timing in tests, we already `setTestNow` to an explicit time in the parent `TestCase` class that this extends: https://github.com/dealerinspire/laravel-operations/blob/master/tests/TestCase.php#L20